### PR TITLE
refactor(core): Reduce boilterplate code in between tests :broom:, and fix the tests in node.js 20 (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
also cleaned up some imports.

### CODE REVIEW

File: packages/cli/test/integration/nodes.api.test.ts [hunk 1]
```diff
@@ -64,10 +62,6 @@ beforeEach(async () => {
 	mocked(findInstalledPackage).mockReset();
 });
 
-afterAll(async () => {
-	await testDb.terminate();
```
Comment (krynble) [inline packages/cli/test/integration/nodes.api.test.ts:68]: Is this handled by the clean up process of the test?

**Review by krynble — APPROVED** [review]
I had a general look, everything looks all right. I saw we removed a couple calls to `testDb.terminate()` so I'm assuming it's not necessary, but I left a comment with this question anyway. Feel free to disregard if this doesn't apply.

File: packages/cli/test/integration/nodes.api.test.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/test/integration/nodes.api.test.ts:68]: This moved [here](https://github.com/n8n-io/n8n/pull/6654/files#diff-d9cac8044aada927b292ffc11b0cd962d808f27c08f220c91e552ce0a5576cddR270), so we don't need to keep repeating this in every test.

**Comment by janober** [comment]
Got released with [`n8n@1.1.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.1.0)

**Pull request merged**